### ISSUE TITLE
Author string doesnt support optional email/url when using shorthand

### ISSUE THREAD

**Issue by pdehaan** [michaelfaith/package-json-validator#12]
I think this should be valid, but it is complaining:

```
{
  "name": "foo",
  "author": "peter",
  "version": "0.0.1"
}
```

The error I'm getting from the online validator is:

```
{
  "valid": false,
  "errors": [
    "String not valid for author, expected format is Barney Rubble <b@rubble.com> (http://barnyrubble.tumblr.com/)"
  ],
  "warnings": [
   ...
  ],
  "recommendations": [
    ...
  ]
}
```
